src/lib/components/comparisonTable/hooks/useComparisonTable.ts

```diff
@@ -110,8 +112,21 @@ export const useComparisonTable = () => {
     setShowMore(!showMore);
   };
 
+  useEffect(() => {
+    if (headerRef.current) {
+      return;
+    }
+
+    const headerById = document.getElementById(headerId);
```

Comment:
The question is, what will happen if you use two of them? The headerId would be the same in that case, right?

Reply:
Ah, true, great catch! 
I started by using `useId` but our react version doesn't have that yet. I'll use our own `generateId` function here

Reply:
Fixed 🔨